l10n_nl_partner_salutation/models/res_partner.py

```diff
@@ -47,67 +47,63 @@ def get_salutation_name_format(self):
     def get_salutation_name(self):
         """Return the name formatted for use in salutation."""
         self.ensure_one()
-        # Use API from partner_firstname
-        # (actually method as it has been overridden in partner_name_dutch)
+        # Use _get_computed_name from partner_firstname.
+        # (actually method as it has been overridden in l10n_nl_partner_name_dutch).
         return self.with_context(
             name_format=self.get_salutation_name_format()
         )._get_computed_name(
             self.lastname, self.firstname, initials=self.initials,
             infix=self.infix
         )
 
-    @api.depends(
-        'gender',
-        'title',
-        'use_manual_salutations',
-        'salutation_manual',
-        'salutation_address_manual'
-    )
```

Comment:
Even though this is not a stored computed field, I was under the impression that it still needs a "depends" clause so that it can live-update in a form view in the way that "onchange" does. Is that true or am I mistaken?

Reply:
Agreed. What do you think @NL66278?

Reply:
Agreed also. I added the depends back in.